salt/utils/network.py

```diff
@@ -1006,9 +1005,6 @@ def win_interfaces():
     '''
     Obtain interface information for Windows systems
     '''
-    if WIN_NETWORK_LOADED is False:
-        # Let's throw the ImportException again
-        import salt.utils.win_network as _
     return salt.utils.win_network.get_interface_info()
```

Comment:
Just `import salt.utils.win_network` here..... There's a pending PR for these changes. https://github.com/saltstack/salt/pull/53496/files